lib/sisimai/bite/email/exim.rb

```diff
@@ -0,0 +1,522 @@
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Exim.pm
+      require 'sisimai/bite/email'
+
+      ReE = {
+        :from    => %r/[@].+[.]mail[.]ru[>]?/,
+      }
+      Re0 = {
+        :from    => %r/\AMail Delivery System/,
+        :subject => %r{(?:
+           Mail[ ]delivery[ ]failed(:[ ]returning[ ]message[ ]to[ ]sender)?
+          |Warning:[ ]message[ ].+[ ]delayed[ ]+
+          |Delivery[ ]Status[ ]Notification
+          |Mail[ ]failure
+          |Message[ ]frozen
+          |error[(]s[)][ ]in[ ]forwarding[ ]or[ ]filtering
+          )
+        }x,
+        # :'message-id' => %r/\A[<]\w+[-]\w+[-]\w+[@].+\z/,
+        # Message-Id: <E1P1YNN-0003AD-Ga@example.org>
+      }
+
+      # Error text regular expressions which defined in exim/src/deliver.c
+      #
+      # deliver.c:6292| fprintf(f,
+      # deliver.c:6293|"This message was created automatically by mail delivery software.\n");
+      # deliver.c:6294|        if (to_sender)
+      # deliver.c:6295|          {
+      # deliver.c:6296|          fprintf(f,
+      # deliver.c:6297|"\nA message that you sent could not be delivered to one or more of its\n"
+      # deliver.c:6298|"recipients. This is a permanent error. The following address(es) failed:\n");
+      # deliver.c:6299|          }
+      # deliver.c:6300|        else
+      # deliver.c:6301|          {
+      # deliver.c:6302|          fprintf(f,
+      # deliver.c:6303|"\nA message sent by\n\n  <%s>\n\n"
+      # deliver.c:6304|"could not be delivered to one or more of its recipients. The following\n"
+      # deliver.c:6305|"address(es) failed:\n", sender_address);
+      # deliver.c:6306|          }
+      #
+      # deliver.c:6423|          if (bounce_return_body) fprintf(f,
+      # deliver.c:6424|"------ This is a copy of the message, including all the headers. ------\n");
+      # deliver.c:6425|          else fprintf(f,
+      # deliver.c:6426|"------ This is a copy of the message's headers. ------\n");
+      Re1 = {
+        :alias  => %r/\A([ ]+an[ ]undisclosed[ ]address)\z/,
+        :frozen => %r/\AMessage .+ (?:has been frozen|was frozen on arrival)/,
+        :rfc822 => %r{\A(?:
+             [-]+[ ]This[ ]is[ ]a[ ]copy[ ]of[ ]the[ ]message.+headers[.][ ][-]+
+            |Content-Type:[ ]*message/rfc822
+            )\z
+          }x,
+        :begin  => %r{\A(?>
+           This[ ]message[ ]was[ ]created[ ]automatically[ ]by[ ]mail[ ]delivery[ ]software[.]
+          |A[ ]message[ ]that[ ]you[ ]sent[ ]was[ ]rejected[ ]by[ ]the[ ]local[ ]scanning[ ]code
+          |Message[ ].+[ ](?:has[ ]been[ ]frozen|was[ ]frozen[ ]on[ ]arrival)
+          |The[ ].+[ ]router[ ]encountered[ ]the[ ]following[ ]error[(]s[)]:
+          )
+         }x,
+        :deliverystatus => %r|\AContent-type: message/delivery-status|,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+
+      ReCommand = [
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->